src/blockchain_utilities/blockchain_ancestry.cpp

```diff
@@ -482,8 +481,7 @@ int main(int argc, char* argv[])
                 }
                 else
                 {
-                  const crypto::hash block_hash = db->get_block_hash_from_height(od.height);
-                  cryptonote::blobdata bd = db->get_block_blob(block_hash);
+                  cryptonote::blobdata bd = db->get_block_blob_from_height(od.height);
```

Comment:
```suggestion
                  const cryptonote::blobdata bd = db->get_block_blob_from_height(od.height);
```

Reply:
Since this isn't a ref or pointer, there's not much point.

Reply:
Is there any point to have it non const?

Reply:
No.